web/src/pages/apd/activities/cost-allocation/CostAllocationRows.js

```diff
@@ -44,32 +59,47 @@ const CostAllocationRows = ({ years, ffy, otherFunding, activityIndex }) => (
       <tr className="budget-table--row__header">
         <th scope="col">State Staff</th>
         <th scope="col" colSpan="2">
-          <span className="sr-only">unit cost</span>
+          <span className="ds-u-visibility--screen-reader">unit cost</span>
         </th>
         <th scope="col" colSpan="2">
-          <span className="sr-only">units</span>
+          <span className="ds-u-visibility--screen-reader">units</span>
         </th>
         <th scope="col">Total cost</th>
       </tr>
     )}
-    <CostSummaryRows items={years[ffy].keyPersonnel} />
-    <CostSummaryRows items={years[ffy].statePersonnel} />
+    <CostSummaryRows
+      items={years[ffy].keyPersonnel}
+      highlightSubtotals={highlightSubtotals}
+    />
+    <CostSummaryRows
+      items={years[ffy].statePersonnel}
+      highlightSubtotals={highlightSubtotals}
+    />
     {otherFunding && (
       <tr>
         <td className="title" colSpan="4">
           Other Funding Amount
         </td>
         <td>-</td>
-        <td className="budget-table--number">
+        <td
+          className={`budget-table--number ${
+            highlightSubtotals ? 'budget-table--cell__hightlight-lighter' : ''
```

Comment:
This is a clever way to handle adding a class. One thing I've used on other projects (and it looks like it's installed in our project already) is this [classnames](https://www.npmjs.com/package/classnames) package. It's a nice way to handle dynamic classes.

You can set up an object for your classes with whatever simple or crazy logic you need for each style class, like:
```
const budgetTableClassName = classNames({
  'budget-table--number': true,
  'budget-table--cell__hightlight-lighter': highlightSubtotals,
  'some-rando-style': shouldShowForA || shouldShowForB
})
```
Then you can just reference your variable as the class and all the logic is prehandled.
`<td className={budgetTableClassName}>`
It's pretty neat!

Reply:
Ohhh I like that. Will update